src/07-projects/project-items.apib

```diff
@@ -40,6 +40,7 @@ Get a list of project items.
                 + (object)
                     + field: `occurred_at`
                     + order: `desc`
+        + withTotals: false (boolean, optional)
```

Comment:
Q: why is this camelCased? Is this a convention or should it be snake_cased?

Reply:
Very good point!